File: code/MethodReturn.java
class MethodReturn {
  public static void main(String[] args) {
    MethodReturn obj = new MethodReturn();
    System.out.println(obj.divide(0, 10));
    System.out.println(obj.divide(10, 5));
  }

  public int divide(int a, int b) {
    if (a == 0)
      return 0;
    return a / b;
  }
}
// 0
// 2

File: code/MethodReturn.php
<?php

function divide(int $a, int $b) : float
{
    if ($a == 0)
        return 0;
    return $a / $b;
}

echo divide(0, 10) . "\n";
echo divide(10, 5) . "\n";
echo divide(5, 10) . "\n";
